scylla-housekeeping

```diff
@@ -1,4 +1,5 @@
-#!/usr/bin/python2
+#!/usr/bin/env python3
```

Comment:
Hi @ultrabug , this introduce a problem (only exists on CentOS, deb like distros are good): 

```
| [stderr] Traceback (most recent call last):
| [stderr]   File "/usr/lib/scylla/scylla-housekeeping", line 33, in <module>
| [stderr]     from pkg_resources import parse_version
| [stderr] ImportError: No module named 'pkg_resources'
```

I solved the problem by installing python34-setuptools-19.6.2-3.el7.noarch
I had asked @amnonh / @syuu1228 to update dist/redhat/scylla.spec.mustache.

Reply:
Sent a patch to scylladb-dev list:
[PATCH scylla] redhat/scylla.spec: add python34-setuptools dependency